src/cryptorandomt.ts

```diff
@@ -0,0 +1,11 @@
+ export let getCryptoRandomInt = (max:any) => {      
+    const getRandomValues = require('get-random-values'), 
+    array = new Uint8Array(1);
+    let min=0; 
+    var byteArray = new Uint8Array(1);
+    getRandomValues(byteArray);
+    let randomNum = Number('0.' + byteArray[0].toString());
+    randomNum = Math.floor(Number(randomNum) * (max - min + 1)) + min;
```

Comment:
randomNum is already a number and also `min` is zero never changed so. you can use 
```ts
randomNum = Math.floor(randomNum * (max - 1)) + min;
```